mkt/websites/management/commands/import_games_from_csv.py

```diff
@@ -116,15 +117,19 @@ def create_instances(self, data):
                     # Keywords use a M2M, so do that once the website is saved.
                     self.set_tags(website, row)
 
+                    WebsiteIndexer.index([website.id], no_delay=True)
```

Comment:
I think `index` should be `index_ids` here. `index` method needs a document and an id, and does not take a no_delay argument since it directly invokes ES. `index_ids` does what you want